index.html

```diff
@@ -244,11 +392,12 @@ <h1>
                 Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
                
              </p> 
+             </div>
```

Comment:
- [ ]  Kindly you should fix the margin/padding in the "About me" section, also you can consider using `max-width attribute for the content to avoid this kind of issue.

Yours | Figma design

![image](https://user-images.githubusercontent.com/60164239/130210470-9e15b5ac-7896-4100-a223-652453c164fc.png)